<ui version="4.0" >
 <class>MotorTuning</class>
 <widget class="QWidget" name="MotorTuning" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>213</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>320</width>
    <height>213</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>320</width>
    <height>213</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="horizontalSpacing" >
      <number>0</number>
     </property>
     <property name="verticalSpacing" >
      <number>0</number>
     </property>
     <item rowspan="2" row="0" column="0" >
      <widget class="QPushButton" name="ui_MotorDecButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>40</horstretch>
         <verstretch>40</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string>QPushButton{
icon-size:35px;
border:0px;
}</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../rc/images.qrc" >:/actions/arrowLf-40.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="autoRepeat" >
        <bool>false</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="1" >
      <widget class="QLabel" name="ui_MotorNumberLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>30</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>50</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet" >
        <string>color:black;
font:bold;
font-size:20pt;</string>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="2" >
      <widget class="QPushButton" name="ui_MotorIncButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>40</horstretch>
         <verstretch>40</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string>QPushButton{
icon-size:35px;
border:0px;
}</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../rc/images.qrc" >:/actions/arrowRt-40.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="autoRepeat" >
        <bool>false</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="4" >
      <widget class="QLabel" name="title" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet" >
        <string>color:black;
font:bold;
font-size:20pt;</string>
       </property>
       <property name="text" >
        <string>Motor Tune</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="5" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item rowspan="2" row="0" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QComboBox" name="ui_GainComboBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>100</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ui_Check0" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ui_Check1" >
       <property name="text" >
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ui_Check2" >
       <property name="text" >
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ui_Check3" >
       <property name="text" >
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="horizontalSpacing" >
      <number>3</number>
     </property>
     <property name="verticalSpacing" >
      <number>3</number>
     </property>
     <item row="2" column="3" colspan="4" >
      <widget class="QSlider" name="ui_DivSlider" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>15</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="layoutDirection" >
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet" >
        <string/>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="4" >
      <widget class="QSlider" name="ui_MultSlider" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>15</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="layoutDirection" >
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="minimum" >
        <number>0</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item rowspan="5" row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item rowspan="5" row="0" column="7" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Divider:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_2" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Multiplier:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="ui_MultLabel" >
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLabel" name="ui_DivLabel" >
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QCheckBox" name="ui_NegCheck" >
       <property name="text" >
        <string>Neg. Gain</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Mul/Div = </string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="5" >
      <widget class="QLabel" name="ui_GainValueLabel" >
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="6" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="6" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_TargetSpeedLine" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>40</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ui_TargetPositionLine" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>20</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="ui_ClearButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>80</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet" >
        <string>QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
font:bold;
font-size:12pt;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
font:bold;
font-size:12pt;
}</string>
       </property>
       <property name="text" >
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="font" >
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Position:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ui_MotorPositionLabel" >
       <property name="minimumSize" >
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ui_PlayButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="font" >
        <font/>
       </property>
       <property name="styleSheet" >
        <string>QPushButton{
border:0px;
background-image:url(:/actions/rivet30x30Go.png);
}
QPushButton:checked{
border:0px;
background-image:url(:/actions/rivet30x30Stop.png);
}</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="iconSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/images.qrc" />
 </resources>
 <connections/>
</ui>
